package-lock.json

```diff
@@ -4753,8 +4753,7 @@
 						"ansi-regex": {
 							"version": "2.1.1",
 							"bundled": true,
-							"dev": true,
-							"optional": true
+							"dev": true
```

Comment:
That's surprising. Ideally, there should be no difference in `package-lock.json` file. Which version of npm do you use?

Reply:
I am using 6.4.1

Reply:
Can you remove changes to `package-lock.json` from this PR? You can copy the original lock file from `master` branch.